src/main/java/core/basesyntax/Machines/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.Machines;
+
+public class Bulldozer extends Machine {
+
+  @Override
+  public void doWork() {
+    System.out.println("Bulldozer is bulldozing a building.");
+  }
+  @Override
+  public void stopWork() {
+    System.out.println("Bulldozer is no longer working");
```

Comment:
The stop work message is not informative enough. It should indicate that it is a Bulldozer that has stopped working. Consider changing the message to 'Bulldozer has stopped working' to provide clear context.